data/schema/world.json

```diff
@@ -56,6 +56,10 @@
             "type": "boolean",
             "description": "Whether players are allowed to scroll the world map."
         },
+        "mask": {
```

Comment:
Wait, I'm confused. Is this something we already had, but forgot to add to the schema (and shouldn't that be impossible, since we validate all the scenarios against the schema)? Or is this a new field you're adding in this PR?

Reply:
Use of the `"mask"` property at the toplevel became possible as of #2071.  Eventually it would be nice to re-use the `structure.json` schema as a `$ref` inside the `world.json` schema to avoid duplication.  But I think that may require putting it under a new key, which would entail re-organizing all existing scenario files (probably doable with automation).

Reply:
Ok, I have now moved the schema update to #2085, which will have to be merged before this PR.